waspc/src/Parser/Action.hs

```diff
@@ -3,19 +3,40 @@ module Parser.Action
     ) where
 
 import Text.Parsec.String (Parser)
+import Data.Maybe (listToMaybe, fromMaybe)
 
-import qualified Wasp.Action as Action
+import qualified Wasp.Action
+import qualified Wasp.JsImport
 
-import qualified Parser.Common as Common
-import qualified Parser.JsCode as JsCode
+import qualified Parser.JsImport
+import qualified Parser.Common as C
 import qualified Lexer as L
 
+-- TODO: Right now, this is mostly just duplicated Parser.Query.
```

Comment:
I didn't want to bother generalizing now since it is not much code.

Reply:
I think it is a good decision as well